src/test/java/uk/gov/companieshouse/charges/data/transform/ChargesTransformerTest.java

```diff
@@ -30,7 +34,7 @@ public void shouldTransformPayloadCorrectly(){
         String chargeId = "MzRiNTU3NjNjZWI1Y2YxMzkzYWY3MzQ0YzVlOTg4ZGVhZTBkYWI4Ng==";
         InternalChargeApi requestBody = new InternalChargeApi();
         var internalData = new InternalData();
-        internalData.setDeltaAt(OffsetDateTime.now());
+        internalData.setDeltaAt(OffsetDateTime.now(ZoneOffset.UTC).truncatedTo(ChronoUnit.SECONDS));
```

Comment:
Is it required to truncate to seconds? maybe yes but just curious

Reply:
This was a change suggest by Gianmario when I connected with him. I can try without.